velox/docs/monthly-updates/august-2022.rst

```diff
@@ -0,0 +1,87 @@
+******************
+August 2022 Update
+******************
+
+Documentation
+=============
+
+* Add `website`_ for Velox.
+* Add documentation for :doc:`/functions/window`.
+* Add documentation for :func:`combinations` and :func:`array_sort` Presto functions.
+
+.. _website: https://velox-lib.io
+
+Core Library
+============
+
+* Add support for Right Semi Join.
+* Add basic :ref:`Window operator<window-node>`.
+* Add support for spilling `order by`.
+* Add support for zero-copy `vector view`_.
+* Add support for DECIMAL addition and subtraction functions.
+* Add support for casting one DECIMAL to another DECIMAL with different precision and scale.
+* Add support for DECIMAL sum aggregation.
+* Improve spilling by avoiding redundant computations, better test coverage.
+* Resolve to vector functions over simple functions when their signatures match.
+
+.. _vector view: https://github.com/facebookincubator/velox/discussions/2212
+
+Substrait Extensions
+====================
+
+* Add support for ROW, ARRAY, and MAP types in Substrait to Velox conversion.
+
+
+Arrow Extensions
+================
+
+* Improve Arrow to Velox conversion for ARRAY and MAP types, and add support for dictionary.
```

Comment:
We can drop "Arrow to Velox conversion". We can also split this update into two and put "Add" before "Improve".

* Add support for dictionary encoding.
* Improve support for ARRAY and MAP types.